test/sol6/stakingFuzzTests.js

```diff
@@ -0,0 +1,66 @@
+const TestToken = artifacts.require("TestToken.sol");
+// using mock contract here, as we need to read the hasInited value
+const MockKyberStaking = artifacts.require("MockStakingContract.sol");
+
+const Helper = require("../helper.js");
+const BN = web3.utils.BN;
+const StakeSimulator = require("../../scripts/stakingSimulator.js");
+const { precisionUnits } = require("../helper.js");
+
+//global variables
+//////////////////
+const NUM_RUNS = 20;
+
+// accounts
+let admin;
+let daoSetter;
+let stakers;
+
+// token
+let kncToken;
+const tokenDecimals = 18;
+
+// staking and its params
+let kyberStaking;
+let epochPeriod = new BN(1000);
+let firstBlockTimestamp;
+
+contract('KyberStaking simulator', async (accounts) => {
+    before('one time init: Stakers, KyberStaking, KNC token', async() => {
+        admin = accounts[1];
+        daoSetter = accounts[2];
+        stakers = accounts.slice(5,); // 5 stakers
+        kncToken = await TestToken.new("kyber Crystals", "KNC", tokenDecimals);
+
+        // prepare kyber staking
+        firstBlockTimestamp = await Helper.getCurrentBlockTime();
+
+        kyberStaking = await MockKyberStaking.new(
+            kncToken.address,
+            epochPeriod,
+            firstBlockTimestamp + 10,
```

Comment:
why only 10? seems like it will never have any actions at epoch 0 :/

Reply:
oh, so should increase this, kay